src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Properties/InterceptedProjectProperties/BuildPropertyPage/DefineConstantsVBValueProvider.cs

```diff
@@ -0,0 +1,31 @@
+﻿// Licensed to the .NET Foundation under one or more agreements. The .NET Foundation licenses this file to you under the MIT license. See the LICENSE.md file in the project root for more information.
+
+using Microsoft.VisualStudio.ProjectSystem.Debug;
+
+namespace Microsoft.VisualStudio.ProjectSystem.Properties;
+
+[ExportInterceptingPropertyValueProvider(DefineConstantsPropertyName, ExportInterceptingPropertyValueProviderFile.ProjectFile)]
+[AppliesTo(ProjectCapability.VisualBasic)]
+internal sealed class DefineConstantsVBValueProvider : InterceptingPropertyValueProviderBase
+{
+    internal const string DefineConstantsPropertyName = "DefineConstants";
+
+    private readonly NameQuotedValuePairListEncoding _encoding = new();
```

Comment:
nit> instantiate from constructor

Reply:
I prefer initializing here, especially when there is no constructor. I don't know that we have a standard throughout the code though.